<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="mvibot">
<xacro:property name="width" value="0.69" />
<xacro:property name="lenght" value="0.813" />
<xacro:property name="height" value="0.3845" />
<xacro:property name="R" value="0.07" />
<xacro:property name="L" value="0.485" />
<xacro:arg name="mvibot_seri" default="MB22_916b"/>
  <!--color robot ######################## (139 139 131)	-->
  <material name="robot_color">
    <color rgba="${0/255} ${255/255} ${255/255} 0.5"/>
  </material>
  <material name="robot_color2">
    <color rgba="${255/255} ${255/255} ${0/255} 0.9"/>
  </material>
  <material name="white">
    <color rgba="1 1 1 1"/>
  </material>
  <material name="white1">
    <color rgba="0 1 0 1"/>
  </material>

<!--link robot ########################-->
  <!--<link name="$(arg mvibot_seri)/base_footprint" >
    </link> -->

  <link name="$(arg mvibot_seri)/base_footprint">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0" />
      <geometry>
          <!-- <mesh filename="file:///home/hitech/catkin_ws/src/rosmvibot/param/ABC7.stl" scale="0.001 0.001 0.001" /> -->
          <!-- <mesh filename="ABC7.stl" scale="0.001 0.001 0.001" /> -->
          <mesh filename="mvibot5.stl" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="robot_color"/>
    </visual>
     <visual>
      <origin rpy="0 0 0" xyz="0 0 0.41" />
      <geometry>
          <!-- <mesh filename="file:///home/hitech/catkin_ws/src/rosmvibot/param/ABC7.stl" scale="0.001 0.001 0.001" /> -->
          <!-- <mesh filename="ABC7.stl" scale="0.001 0.001 0.001" /> -->
          <mesh filename="dir3.stl" scale="0.001 0.001 0.001"/>
      </geometry>
      <material name="robot_color2"/>
    </visual>
  </link>
</robot>
